sycl/doc/SYCLCompilerAndRuntimeDesign.md

```diff
@@ -217,17 +217,16 @@ and understands mnemonics designating a particular code form, for example
 architecture).  User can specify desired code format using the target-specific
 option mechanism, similar to OpenMP.
 
-`-Xsycl-target=<triple> <arg>`
+`-Xsycl-target-backend=<triple> "<arg>,<arg>"`
```

Comment:
Argument is "<arg> <arg>" (not csv)

Reply:
Why there are two args? There can be any number of them

Reply:
This was intended as "any number of args". Will fix the ambiguity and the format.